instructor-notes/stacking.md

```diff
@@ -31,12 +33,16 @@ This first part of the demonstration is based around one issue with two related
   * Also note that the branch is no longer up-to-date with `main`.
   Explain two ways to handle this (but ultimately do the first one):
     * Within GitHub, merge `main` into `<username>/<issue #>-build-penguin-model`
-    * Within your local repo, checkout `main`, pull down, checkout `<username>/<issue #>-build-penguin-model`, merge on command line, and then push.
+    * Locally, you can checkout `main`, pull down, checkout `<username>/<issue #>-build-penguin-model`, merge on command line, and then push.
 * The second PR can now be approved and merged into `main`.
 
 
 ### Part 2: How to approximate stacking when you are working in a fork
 
+> The second part of the demonstration is based around a different issue with two related tasks:
+>   * Task 1: Add a function to the `utils.R` script that, given a data frame and two numeric variables, exports a scatterplot
+>   * Task 2: Modify the script `model-penguins.R` to make and export a scatterplot of variables used in the regression
+
```

Comment:
I think something bad™️  may have happened with the merge?

Reply:
eh https://github.com/AlexsLemonade/2023-chop-training/pull/71/commits/9d18ab557a8c760207a7fc150b69b3935129528a?